lightning/src/ln/channelmanager.rs

```diff
@@ -368,6 +368,10 @@ pub struct ChannelManager<ChanSigner: ChannelKeys, M: Deref, T: Deref, K: Deref,
 	channel_state: Mutex<ChannelHolder<ChanSigner>>,
 	our_network_key: SecretKey,
 
+	/// Used to track the last value sent in a node_announcement "timestamp" field. We just set
+	/// them to be monotonically increasing since we don't assume access to a time source.
+	last_node_announcement_serial: AtomicUsize,
```

Comment:
rephrase the last sentence to: "We ensure this value increases monotonically, since we don't assume access to a time source."